Userland/Services/WindowServer/WindowManager.cpp

```diff
@@ -172,6 +172,20 @@ void WindowManager::set_scroll_step_size(unsigned step_size)
     m_config->sync();
 }
 
+void WindowManager::set_double_click_speed(int speed)
+{
+    VERIFY(speed >= double_click_speed_min && speed <= double_click_speed_max);
```

Comment:
This allows anyone to crash the window server by connecting to it and setting an out-of-range doubleclick speed.
Instead of `VERIFY`, we should validate the value and reject invalid ones.

Reply:
Oh wait nevermind, you already do this in the ClientConnection handler :)